### PR TITLE
Feat: Lists

### PR DESCRIPTION
**Pull request by RahulSingh9131**
1. normal list
normalList
2. notification page list
NotificationList

### CODE REVIEW

File: css/components/list.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+.page-list h3{
+    color: var(--text-color-dark);
+    font-size: var(--h3-font-size) ;
+}
+.page-list p{
+    color: var(--text-color-light);
+    font-size: var(--normal-font-size);
+}
+.page-list i{
+    margin: 0rem 1rem;
```
Comment (Sameony) [inline css/components/list.css]: you are using variables for specifying margin size above, but rem here. to attain consistency, you may want to use variables here as well.

**Review by Sameony — COMMENTED** [review]
Clean and consistent code.
bow shadow property might be used in other components as well such as in cards, so you can create a utility class for it. If the ratio is different there, then it's ok to merge.

**Review by SimranjitKaurMaan — APPROVED** [review]
lgtm.

File: css/components/list.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
+/* === css for List === */
+
+/* === css for normal list === */
+.list{
+    box-shadow: 0px 5px 10px rgba(0, 0, 0, 0.6);
```
Comment (SimranjitKaurMaan) [inline css/components/list.css:5]: You can use rem here as well. Just a suggestion, not a blocker.

File: css/components/list.css [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    padding: .9rem  1.2rem;
+}
+.list-text{
+    font-size: var(--h3-font-size);
+    border-bottom: .1rem solid var(--text-color-light);
+    margin-bottom: var(--mb-1);
+    color: var(--primary);
```
Comment (SimranjitKaurMaan) [inline css/components/list.css]: You can be more consistent with your naming. color name : `--text-color-light` and `primary` doesn't have consistency.

File: css/components/list.css (hunk 1 above)
Comment (RahulSingh9131) [reply css/components/list.css]: done

File: css/components/list.css (hunk 3 above)
Comment (RahulSingh9131) [reply css/components/list.css]: sure!!! done with the suggestion

**Review by Sameony — APPROVED** [review]

**Pull request merged**